frontend/javascripts/oxalis/view/settings/dataset_settings_view.js

```diff
@@ -136,6 +138,34 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps> {
     );
   };
 
+  getDeleteButton = (layerName: string) => (
+    <Tooltip title="Unlink dataset's original segmentation layer">
+      <Icon
+        type="stop"
+        onClick={() => {
+          this.removeFallbackLayer(layerName);
+        }}
+        style={{
+          position: "absolute",
+          top: 4,
+          right: 26,
+          cursor: "pointer",
+        }}
+      />
+    </Tooltip>
+  );
+
+  removeFallbackLayer = (layerName: string) => {
+    Modal.confirm({
+      title: messages["tracing.confirm_remove_fallback_layer.title"],
+      content: messages["tracing.confirm_remove_fallback_layer.content"],
+      onOk: async () => {
```

Comment:
The modal looks quite packed now with the longer text. I think, you can pass a width parameter here so that the modal gets a bit wider (e.g., `width: 600` could work).
Also, splitting the content into two paragraphs would probably be a good idea, too. Since the `messages` file doesn't contain react elements, I'd suggest to just split the `content` property into two parts. So, the usage would look like this:

```
content: <div>
  <p>messages["tracing.confirm_remove_fallback_layer.content_explanation"]</p>
  <p>messages["tracing.confirm_remove_fallback_layer.content_notes"]</p>
</div>
```